### DISCUSSION TITLE
How to add IntelliSense in unsupported languages?

### DISCUSSION THREAD

**Discussion by ffcabbar** [suren-atoyan/monaco-react#267] [Q&A] [answered]
How to add IntelliSense in a default Language SQL editor?

**Comment by ffcabbar** [comment, marked as answer]
@suren-atoyan Do you have any advice for me? I just want to add IntelliSense for SQL language.

**Reply by suren-atoyan** [reply]
Hello @ffcabbar, sorry for the late answer.

you can use [completion-provider](https://microsoft.github.io/monaco-editor/playground.html#extending-language-services-completion-provider-example) to improve overall language interaction, but in general, this question should have been referred to [monaco-editor](https://github.com/microsoft/monaco-editor), not to `monaco-react` (the last one is just a `React` wrapper around `monaco-editor`)

**Reply by ffcabbar** [reply]
Thank you so much. It works 👍
